docs/developer-docs/gas-cost.md

```diff
@@ -4,15 +4,143 @@
+To get a rough estimate of how much your project may cost, below are a few common project architectures and their estimated monthly cost in cycles. For detailed breakdowns of each cycles cost, please refer to the chart below in [Details: Cost of compute and storage transactions on the Internet Computer](#details-cost-of-compute-and-storage-transactions-on-the-internet-computer).
 
-In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launched with a replication factor in the order of 30, different sizes may become available in the future. Cycles prices for the new high-replication subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets. The pricing mechanics for the Bitcoin Mainnet API is slightly different, see the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works.md) for details.
+:::info
+The estimates below are simply for demonstrating what different sample architectures may cost. The actual cost of your project will vary based on the exact number of resources and canister calls that are used. Therefore, the estimates below should only be used for gaining an idea of what a project may cost, but should not be used for budgeting exact costs. For exact costs, you can calculate using the chart below for more details: [Details: Cost of compute and storage transactions on the Internet Computer](#details-cost-of-compute-and-storage-transactions-on-the-internet-computer).
+:::
+
+:::info
+These estimates use a 13-node subnet. Costs will be different if deployed on a 34-node subnet. Please refer to the chart below for more details: [Details: Cost of compute and storage transactions on the Internet Computer](#details-cost-of-compute-and-storage-transactions-on-the-internet-computer).
+:::
+
+### Single canister
+
+For a single canister that provides a service or function that is called by other canisters through inter-canister calls the following approximate cycles costs would be incurred:
+
+- Canister creation (one-time fee charged when the canister is deployed): 100_000_000_000 cycles.
+
+- Storing 2GB of data: 254_000 cycles per second; 667_585_152_000 per month.
+
+- Using 0.5% compute resources: 5_000_000 cycles per second; 13_141_440_000_000 cycles per month.
```

Comment:
Canister do not use compute because it is too expensive. They rely on best-effort scheduling. The compute is expensive because it is scarce.